src/Sylius/Behat/Context/Api/Shop/AddressContext.php

```diff
@@ -115,6 +220,91 @@ public function iShouldBeNotifiedThatAddressHasBeenDeleted(): void
         Assert::true($this->responseChecker->isDeletionSuccessful($this->client->getLastResponse()));
     }
 
+    /**
+     * @Then I should be notified that the address has been successfully added
+     */
+    public function iShouldBeNotifiedThatTheAddressHasBeenSuccessfullyAdded(): void
+    {
+        Assert::true($this->responseChecker->isCreationSuccessful($this->client->getLastResponse()));
+    }
+
+    /**
+     * @Then /^(address "[^"]+", "[^"]+", "[^"]+", "[^"]+", "[^"]+"(?:|, "[^"]+")) should(?:| still) be marked as my default address$/
+     */
+    public function addressShouldBeMarkedAsMyDefaultAddress(AddressInterface $address): void
+    {
+        $response = $this->responseChecker->getResponseContent($this->client->getLastResponse());
+
+        /** @var CustomerInterface $customer */
+        $customer = $this->iriConverter->getItemFromIri($response['customer']);
```

Comment:
You should check if this address is marked as default in API response, not by comparing with default address from customer entity